### PR TITLE
Retry default gw retrieval on error

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

**What this PR does / why we need it**:
Sometimes it takes some time for DHCP to assign default gw, so
the default gw ping probe fails, here we add a 30 seconds timeout
with a 1 second interval.
**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/helper/client.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ var (
 
 const nmstateCommand = "nmstatectl"
 const vlanFilteringCommand = "vlan-filtering"
+const defaultGwRetrieveTimeout = 30
```
Comment (phoracek) [inline pkg/helper/client.go]: I can imagine the GW reachability in the next constant may be limited by this one. Let's make all of them 120 for now and later we can pass Context around with shared timeout. What do you think?

**Comment by phoracek** [comment]
/approve
/lgtm
/hold

Let's wait for the CI.

**Comment by phoracek** [comment]
We ping `""` now

File: pkg/helper/client.go (hunk 1 above)
Comment (qinqon) [reply pkg/helper/client.go]: 120 is good for me.

**Comment by phoracek** [comment]
@qinqon please rebase

**Comment by qinqon** [comment]
@phoracek let's wait for CI I will rebase

**Comment by qinqon** [comment]
/hold cancel

ci is good.

**Pull request merged**